ext/openssl/tests/session_meta_capture.phpt

```diff
@@ -49,6 +49,11 @@ $clientCode = <<<'CODE'
     @stream_socket_client($serverUri, $errno, $errstr, 2, $clientFlags, $clientCtx);
     $meta = stream_context_get_options($clientCtx)['ssl']['session_meta'];
     var_dump($meta['protocol']);
+
+    stream_context_set_option($clientCtx, 'ssl', 'crypto_method', STREAM_CRYPTO_METHOD_TLSv1_3_CLIENT);
```

Comment:
this will fail if run with older OpenSSL version than 1.1.1

Reply:
yes, what do you think we should do here?
1. Skipping the test if < 1.1.1 would not make much sense.
2. Revert to original ?
3. or?

Reply:
Added new test: cbda625